.travis.osx.before_install.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
As mentioned in person, might be nicer to have this as a script in `build` (it's not gclient-related, but it *is* build-related!) than as a dot file?